### PR TITLE
Azure Backup Health

### PR DESCRIPTION
**Pull request by lapradh**
Updated health details

openapi spec review PR : Azure/azure-rest-api-specs#9711
Check in the metadata TXT file generated when you run generate.ps1(details) : Done
Update api tag in AzSdk.RP.props, release notes in csproj : updated release notes.
Update versions in csproj and assemblyinfo.cs if you are going to release : no new version. so not required.

### CODE REVIEW

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @lapradh , could you
- Add a link to the openapi spec review PR here
- Check in the metadata TXT file generated when you run `generate.ps1`[(details)](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec)
- Update api tag in `AzSdk.RP.props`, release notes in csproj
- Update versions in csproj and `assemblyinfo.cs` if you are going to release

Thanks

**Comment by lapradh** [comment]
Swagger PR : https://github.com/Azure/azure-rest-api-specs/pull/9711

File: sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/Microsoft.Azure.Management.RecoveryServices.Backup.csproj [hunk 1]
```diff
@@ -12,6 +12,10 @@
 		<PackageReleaseNotes>
 			<![CDATA[
 New features:
+- Added changes for Azure Backup Health.
+       ]]>
+			<![CDATA[
+New features:
 - Added 4 new rest api related to private endpoint connection work (JEDI).
        ]]>
 		</PackageReleaseNotes>
```
Comment (isra-fel) [inline sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/Microsoft.Azure.Management.RecoveryServices.Backup.csproj:21]: @lapradh this looks like a bad merge. There should be one `<![CDATA[` `]]>` block within `<PackageReleaseNotes>`.

**Comment by isra-fel** [comment]
And please also resolve the conflicts, as another PR of recovery service backup is merged #12650

File: sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/Microsoft.Azure.Management.RecoveryServices.Backup.csproj (hunk 1 above)
Comment (lapradh) [reply sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/Microsoft.Azure.Management.RecoveryServices.Backup.csproj]: Pls review now

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**